index.html

```diff
@@ -295,6 +295,45 @@ <h2>Welcome to MyPetSpotlight!</h2>
         </div>
       </div>
     </div>
+    
+    <div class="card">
+      <img src="./assets/images/littlemike.jpg" alt="description of your picture">

```

Comment:
Looks good but please replace `"description of your picture"` with an actual description of your picture like "Picture of the Little Mike"

Reply:
Thank you, fixed! 💪